### ISSUE TITLE
How tp buy license without google services

### ISSUE THREAD

**Issue by titilambert** [firebirdberlin/nightdream#179]
Hello !
I'm trying to convert my old Nexus 5 to a powerfull BedSide Alarm Clock.
Thanks to your apply this is be really easy !
I would to buy a license but nothing happens when I click on the link in the application. The reason, I guess, it's because I use lineageOS without the Google Services...
Do you have any other way to get a License ?
Thanks !

**Comment by firebirdberlin** [comment]
Do you have a smart idea how this could be achieved ? How did you install the app if not via Google Play ?

**Comment by titilambert** [comment]
I installed it useing apkpure: https://apkpure.com/day-and-night-clock/com.firebirdberlin.nightdream
Can I pay you directly through paypal ?
BTW, if your software is open source, I would suggest you to publish your app on Fdroid: https://f-droid.org/en/

**Comment by martin3000** [comment]
same here. since the google company is no longer on the good side, I stop working with google. I install apps via F-Droid and Aptoide.

**Comment by firebirdberlin** [comment]
I totally agree but the number of F-droid users is relatively small compared to users with the Play store installed. I would have to maintain different market versions with a high effort. I just don't think this step will pay off.

**Comment by martin3000** [comment]
Also there is no google play store in china.

**Comment by fullcavediver** [comment]
I would also like to buy the app without google services. The app runs perfectly on an old cell phone without a SIM card, Wi-Fi and Google and replace a classic night clock. Unfortunately currently only in the open source version. Thank you!
